### PR TITLE
Open zapplin update

### PR DESCRIPTION
**Pull request by 0xAshish**
WIP: please review and keep it open.

### CODE REVIEW

File: test/child/ChildERC20.test.js [hunk 1]
```diff
@@ -62,18 +66,22 @@ contract('ChildERC20', async function(accounts) {
...
-    receipt.logs[1].args.amount.should.be.bignumber.equal(amount)
-    receipt.logs[1].args.output1.should.be.bignumber.equal(0)
+    receipt.logs.should.have.lengthOf(3)
+
+    receipt.logs[0].event.should.equal('Transfer')
+    receipt.logs[0].args.from.toLowerCase().should.equal(owner)
+    receipt.logs[0].args.to.toLowerCase().should.equal(ZeroAddress)
+    receipt.logs[0].args.value.should.be.bignumber.equal(amount)
+
+    receipt.logs[1].event.should.equal('Withdraw')
```
Comment (jdkanani) [inline test/child/ChildERC20.test.js]: Remove `Withdraw` and use `Transfer` only.

Comment (0xAshish) [reply test/child/ChildERC20.test.js]: we are not getting that token address in transfer event so, will keep withdraw event for now and optimize on sending params part and just send only withdraw event logs.

File: test/proofs/ExitValidator.test.js [hunk 2]
```diff
@@ -91,15 +91,20 @@ contract('ExitValidator', async function(accounts) {
...
+        )
+        childToken = ChildToken.at(receipt.logs[1].args.token.toLowerCase())
 
         // exit validator
         exitValidator = await ExitValidator.new()
         rootChain = await RootChainMock.new(rootToken.address) // dummy address for stakemanager
         depositManager = await DepositManagerMock.new({ from: owner })
         withdrawManager = await WithdrawManagerMock.new({ from: owner })
-
+
```
Comment (jdkanani) [inline test/proofs/ExitValidator.test.js]: Remove trailing space!

**Comment by jdkanani** [comment]
Please, remove dead code

**Pull request merged**